src/index.js

```diff
@@ -10,6 +10,9 @@ if (typeof window !== 'undefined' && window.Vue) {
   window.Vue.use(plugin)
 }
 
+// Allow doing VueMotion.presets.custom = ...
+plugin.presets = presets
```

Comment:
No. I meant that I should not have missed adding this line so there should probably be a test to cover its existence.